src/benches/eth_erc20.rs

```diff
@@ -1,20 +1,22 @@
-use aurora_engine::parameters::SubmitResult;
-use aurora_engine::prelude::{Address, U256};
-use aurora_engine::transaction::EthTransaction;
+use crate::parameters::SubmitResult;
+use crate::prelude::{Address, U256};
+use crate::transaction::EthTransaction;
 use borsh::BorshDeserialize;
-use criterion::{criterion_group, BatchSize, BenchmarkId, Criterion};
+use criterion::{BatchSize, BenchmarkId, Criterion};
 use near_vm_logic::VMOutcome;
 use secp256k1::SecretKey;
 use std::path::{Path, PathBuf};
 
-use super::{address_from_secret_key, deploy_evm, sign_transaction, AuroraRunner, SUBMIT};
-use crate::solidity;
+use crate::test_utils::solidity;
+use crate::test_utils::{
+    address_from_secret_key, deploy_evm, sign_transaction, AuroraRunner, SUBMIT,
+};
 
 const INITIAL_BALANCE: u64 = 1000;
 const INITIAL_NONCE: u64 = 0;
 const TRANSFER_AMOUNT: u64 = 67;
 
-fn eth_erc20_benchmark(c: &mut Criterion) {
+pub fn eth_erc20_benchmark(c: &mut Criterion) {
```

Comment:
This can now be `pub(crate)` I think? In general, I personally have two pronged approach to visibility in Rust:

* I try very hard not to make things `pub` if I can
* I don’t really care if things are private, pub(super) or pub(crate) otherwise


In large multi-crate projects, using pub solely for cross-crate interfaces improves readability a lot.